### PR TITLE
Added roles to profile page

### PR DESCRIPTION
**Pull request by cen10**
Issue #557 

I couldn't get the roles to show up directly underneath the login name when using 'content_for' so I changed it to an h1 instead (I'm not well-versed in haml). Wasn't sure if you envisioned people holding multiple roles, so the list_roles method accounts for that.

This is my first pull request, so your feedback/corrections would be appreciated!

### CODE REVIEW

**Comment by maco** [comment]
What's going on with ```content_for :title``` is that its location is already set in https://github.com/Growstuff/growstuff/blob/dev/app/views/layouts/application.html.haml

I wonder if a string including a ```<br/>``` could go in the ```:subtitle```?

**Comment by cen10** [comment]
Thanks, I'll look into that. I'm getting "unknown type of %string" and "unexpected end of input" syntax errors whenever I try to include anything either on the same line as `content_for :subtitle` or below it using a block.

**Comment by Skud** [comment]
We definitely don't want to mess with the title blocks.  We are trying to standardise the way we display headers (title, subtitle, button bar) across the website and we shouldn't put anything else in that area without some high-level design discussion.

I'd rather these roles just showed in the sidebar somewhere (maybe near the account status and similar).

**Comment by cen10** [comment]
I moved the roles directly above "Member since:" in the sidebar. Let me know what you think.

**Comment by Skud** [comment]
This looks good to me now, but if this is your first commit, can you please add yourself to CONTRIBUTORS.md?  Then I think it's good to merge :)

File: app/models/member.rb [hunk 1]
```diff
@@ -107,6 +107,15 @@ def has_role?(role_sym)
     roles.any? { |r| r.name.gsub(/\s+/, "_").underscore.to_sym == role_sym }
   end
 
+  def list_roles
```
Comment (Skud) [inline app/models/member.rb]: Wait, sorry!  I said I think this is good to merge but I was just looking at the views.

I don't think we need this method here.  Why can't we just use join(", ") here, as we do in many other views?

**Comment by pozorvlak** [comment]
Another attempt to address the same issue: #754.

**Comment by cesy** [comment]
#787 is the latest one related to this issue. Does that mean this pull request should be closed as superseded by that, or is there anything from here that we need to keep/save? Sorry about the delays and confusion, @cen10.

**Comment by pozorvlak** [comment]
I think we should probably close this as superseded by #787. @cen10, thanks for your contribution, and I'm sorry for the delay!

**Pull request closed without merging**